channels.go

```diff
@@ -49,3 +49,43 @@ func (c *Client) SearchChannels(params *SearchChannelsParams) (*SearchChannelsRe
 
 	return channels, nil
 }
+
+// GetChannelInformationParams ...
+type GetChannelInformationParams struct {
+	BroadcasterID string `query:"broadcaster_id"`
+}
+
+// GetChannelInformationResponse ...
+type GetChannelInformationResponse struct {
+	ResponseCommon
+	Data ManyChannelInformation
+}
+
+// ManyChannelInformation ...
+type ManyChannelInformation struct {
+	Channels []ChannelInformation `json:"data"`
+}
+
+// ChannelInformation ...
+type ChannelInformation struct {
+	ID         string `json:"broadcaster_id"`
+	Display    string `json:"broadcaster_name"`
+	Language   string `json:"broadcaster_language"`
+	CategoryID string `json:"game_id"`
+	Category   string `json:"game_name"`
```

Comment:
Please adjust these field names as described below:

```suggestion
	BroadcasterID         string `json:"broadcaster_id"`
	BroadcasterName    string `json:"broadcaster_name"`
	BroadcasterLanguage   string `json:"broadcaster_language"`
	GameID string `json:"game_id"`
	GameName   string `json:"game_name"`
```

Reply:
Reason why I changed it to `CategoryID` and `CategoryName` is due to them releasing the new EventSub API and it returning that same information as `Category`. Might be a sign in the near future they will change it to that on helix.

Reply:
Right, thanks for explaining.However, until that change comes into effect I'd prefer we represent the API as it stands today.

Thanks for making those changes.